### PR TITLE
Add list command

### LINKED ISSUE #14: Add list functionality
As a user, I can view a list of all incomes and expenses, so that I can better track my financials.

### LINKED ISSUE #66: Add list function for Incomes only

### LINKED ISSUE #67: Add list function for Expenses only

### PR DESCRIPTION
**Pull request by YFshadaow**

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM

**Review by NeoMinWei — CHANGES_REQUESTED** [review]
can keep the handleInvalidCategory method in DeleteCashflow Command, I think it looks neater
other than that LGTM

**Review by wwweert123 — APPROVED** [review]
Maybe for the execute function in ListCommand.java you can specify what kind of exception is thrown? This might make it easier for debugging. Eg. FinanicalPlannerException

**Review by ryan1604 — APPROVED** [review]
LGTM

**Review by wwweert123 — APPROVED** [review]
Last PR before milestone 1. We did it guys! LGTM!

**Review by NeoMinWei — APPROVED** [review]

File: src/main/java/seedu/financialplanner/commands/DeleteCashflowCommand.java [hunk 1]
```diff
@@ -38,6 +38,13 @@ public DeleteCashflowCommand(RawCommand rawCommand) throws IllegalArgumentExcept
...
+        if (stringCategory != null) {
+            try {
+                category = CashflowCategory.valueOf(stringCategory.toUpperCase());
+            } catch (IllegalArgumentException e) {
+                throw new IllegalArgumentException("Entry must be either income or expense");
+            }
+        }
     }
 
     private void handleInvalidCategory(String stringCategory) {
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/commands/DeleteCashflowCommand.java:50]: thanks for keeping this method

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**